pkg/quarterdeck/users_test.go

```diff
@@ -12,6 +13,70 @@ import (
 	ulids "github.com/rotationalio/ensign/pkg/utils/ulid"
 )
 
+func (s *quarterdeckTestSuite) TestUserDetail() {
+	require := s.Require()
+	defer s.ResetDatabase()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	// Test passing empty ULID results in StatusNotFound error
+	user, err := s.client.UserDetail(ctx, "")
+	s.CheckError(err, http.StatusNotFound, "resource not found")
+	require.Nil(user, "expected no data returned after an error")
+
+	// Test passing invalid ULID results in StatusUnauthorized error
```

Comment:
Should this test be moved after the correct permissions are set? Otherwise I think the error is occurring in the authorization middleware before the code gets to the handler.

Reply:
The comment on what this is testing is misleading.  I will make the change.